From 5f622844605a569eec776f3cdd0954ccefd8c99b Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Sun, 29 Nov 2020 11:55:10 +0100
Subject: [PATCH] Drop unused KF5Plasma dependency

This had been already dropped in commit 1d60f46f, which was reverted in
commit 8ba8a29c. At least that part of it was correct though.

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a945dee..b49bff1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,7 +32,6 @@ find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS
     KDELibs4Support
     DBusAddons
     KCMUtils
-    Plasma
 )
 
 find_package(X11 REQUIRED OPTIONAL_COMPONENTS Xtst)
-- 
2.29.2

